apps/blockchain/lib/blockchain/transaction.ex

```diff
@@ -255,10 +266,18 @@ defmodule Blockchain.Transaction do
 
       # Note, we only want to take the first 3 items from the tuples,
       # as designated Θ_3 in the literature Θ_3
-      {state, remaining_gas_, sub_state_, _output} = Contract.message_call(params)
+      {state, remaining_gas_, sub_state_, status} =
```

Comment:
What does the extra underscore mean in `remaining_gas_` and `sub_state_`? While we're here how would you feel about removing it? If it's a naming collision, how would you feel about extracting a function to call here?

Reply:
I think it's just naming collision. I'll update these to different names. 👍 good catch

Reply:
Updated this in 0ff7ee6ae5abc3a06d32cd9d9d43e36f24ec8fa7